storm-core/src/jvm/backtype/storm/task/TopologyContext.java

```diff
@@ -205,32 +217,65 @@ public Object getTaskData(String name) {
     public void setExecutorData(String name, Object data) {
         _executorData.put(name, data);
     }
-    
+
     public Object getExecutorData(String name) {
         return _executorData.get(name);
-    }    
-    
+    }
+
     public void addTaskHook(ITaskHook hook) {
         hook.prepare(_stormConf, this);
         _hooks.add(hook);
     }
-    
+
     public Collection<ITaskHook> getHooks() {
         return _hooks;
     }
+
+    public Object groupingToJSONableObject(Grouping grouping) {
```

Comment:
Good point. I'll fix that.  I also wasn't sure how much everyone would like my made-up word "JSONable".